src/app/pages/home/profile/profile.page.html

```diff
@@ -6,15 +6,6 @@
         <img ng-show="isFrontCaptured" [src]="card.frontImg" alt=""/>
     </ion-card>
 
-    <ion-card button style="height:172px;" [routerLink]="['/user', user.id]" *ngFor="let user of (users | async)" lines="none">
-      <ion-avatar class="image">
-        <img [src]="frontImg"/>
-      </ion-avatar>
-      <ion-card-header>
-        <ion-card-subtitle>{{ user.name }}</ion-card-subtitle>
-      </ion-card-header>
-    </ion-card>
-
```

Comment:
I think this code bit will be useful when we test populate the rolodex for the infinite loop. You with me?